prime-router/src/main/kotlin/cli/tests/AuthTests.kt

```diff
@@ -1172,107 +1170,84 @@ class Server2ServerAuthTests : CoolTest() {
         )
         val orgEndpoint = "${environment.url}/api/settings/organizations"
 
-        val client = HttpClientUtils.createDefaultHttpClient(
-            userToken
-        )
-
-        val clientAdmin = HttpClientUtils.createDefaultHttpClient(
-            adminToken
-        )
-
         // Case: GET All Org Settings (Admin-only endpoint)
         // Unhappy Path: user on admin-only endpoint
-        val response = runBlocking {
-            client.get(orgEndpoint) {
-                timeout {
-                    requestTimeoutMillis = 45000
-                    // default timeout is 15s; raising higher due to slow Function startup issues
-                }
-                accept(ContentType.Application.Json)
-            }
-        }
+        val response = HttpClientUtils.get(
+            url = orgEndpoint,
+            accessToken = userToken,
+            timeout = 45000, // default timeout is 15s; raising higher due to slow Function startup issues
```

Comment:
Looks like this was there previously but 45 seconds is a very long timeout especially when it comes to tests

Reply:
I don't have context for the comment 

```
// default timeout is 15s; raising higher due to slow Function startup issues
```

but I imagine it's there for a reason. My gutcheck feeling is to leave the value alone but if you feel strongly I'm happy to change it to something else.